### PR TITLE
tomlplusplus: init at 3.4.0

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

File: mingw-w64-tomlplusplus/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "${MINGW_PACKAGE_PREFIX}-meson"
+  "${MINGW_PACKAGE_PREFIX}-cmake"
+)
+provides=(libtomlplusplus.so)
+source=($_realname-$pkgver.tar.gz::https://github.com/marzer/$_realname/archive/refs/tags/v$pkgver.tar.gz)
+sha512sums=('c227fc8147c9459b29ad24002aaf6ab2c42fac22ea04c1c52b283a0172581ccd4527b33c1931e0ef0d1db6b6a53f9e9882c6d4231c7f3494cf070d0220741aa5')
+b2sums=('9495ccd78707ced11744eab7c1c0bf0c0c28e283d186195bb48d1059bae7eb1a874bc964b0fc45210fd73ffd7485ecf3e1159da227d0e1c8ff249e79c08eecf0')
+
+build() {
+  export MESON_PACKAGE_CACHE_DIR="${srcdir}"
```
Comment (ognevny) [inline mingw-w64-tomlplusplus/PKGBUILD:24]: is it needed here?

File: mingw-w64-tomlplusplus/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url="https://marzer.github.io/tomlplusplus/"
+license=(MIT)
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-meson"
+  "${MINGW_PACKAGE_PREFIX}-cmake"
+)
+provides=(libtomlplusplus.so)
```
Comment (Biswa96) [inline mingw-w64-tomlplusplus/PKGBUILD:18]: Did you just copy those things blindly from ArchLinux?

Comment (Freed-Wu) [reply mingw-w64-tomlplusplus/PKGBUILD:18]: Oh, sorry, too mang PRs make I too tired

Comment (Biswa96) [reply mingw-w64-tomlplusplus/PKGBUILD:18]: Take things slowly. There is no deadline or pressure from manager 🙂

Comment (ognevny) [reply mingw-w64-tomlplusplus/PKGBUILD]: > Take things slowly. There is no deadline or pressure from manager 🙂

I should have one, sometimes I do things very slowly...

**Comment by ognevny** [comment]
what's happening to meson? it throws absolutely strange errors

**Comment by Biswa96** [comment]
OP is just keep forgetting to add cc in makedepends.

**Comment by ognevny** [comment]
> OP is just keep forgetting to add cc in makedepends.

and how clang succeeded? I'm now confused

**Comment by Biswa96** [comment]
> and how clang succeeded? I'm now confused

it's clang from Windows SDK. See the log.

```
  Activating VS 17.11.1
  C++ compiler for the host machine: clang++ (clang 17.0.3 "clang version 17.0.3")
  C++ linker for the host machine: clang++ link 14.41.34120.0
```

**Pull request closed without merging**